src/main/java/teammates/ui/webapi/GetFeedbackSessionSubmittedGiverSetAction.java

```diff
@@ -32,11 +32,19 @@ public JsonResult execute() {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
```

Comment:
checkSpecificAccessControl needs to be migrated as well.

Reply:
Migrated as well